dts/bindings/gpio/semtech,sx1509b-gpio.yaml

```diff
@@ -18,6 +18,13 @@ properties:
       required: true
       const: 16
 
+    int-gpios:
```

Comment:
Please name this `nint-gpios` to match the datasheet signal name `NINT`.

Reply:
Done.